wca-regulations.md

```diff
@@ -116,7 +116,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3a1) Competitors must be ready to submit their puzzles when they are called (see [Regulation 2u](regulations:regulation:2u)).
     - 3a2) Puzzles must be fully operational, such that normal scrambling is possible.
     - 3a3) Polyhedral puzzles must use a colour scheme with one unique colour per face in the solved state. Each puzzle variation must have moves, states, and solutions functionally identical to the original puzzle.
-- 3d) Puzzles must have coloured parts, which must be one of the following: coloured stickers, coloured tiles, coloured plastic, or painted/printed colours. All coloured parts of a puzzle must be made of a similar material.
+- 3d) Puzzles must have coloured parts, which define the colour scheme of the puzzle and must be either coloured stickers, coloured tiles, coloured plastic, or painted/printed colours. All coloured parts of a puzzle must be made of a similar material.
```

Comment:
Shouldn't it be "either A or B or C or D"? Or maybe "A, B, C or D"? Though the latter doesn't emphasize that we can't pick several options.

I'm not English native speaker, just found a couple of links:
http://english.stackexchange.com/questions/170409/whats-the-best-way-to-use-either-on-more-than-two-options
http://english.stackexchange.com/questions/40950/is-either-only-used-with-two-options